src/__test__/pages/experiments/[experimentId]/data-processing/index.test.jsx

```diff
@@ -1,41 +1,71 @@
 import React from 'react';
-import { mount, configure } from 'enzyme';
+import { screen, render } from '@testing-library/react';
+import '@testing-library/jest-dom';
+import userEvent from '@testing-library/user-event';
+// import { mount, configure } from 'enzyme';
 import { Provider } from 'react-redux';
 import configureMockStore from 'redux-mock-store';
-import waitForActions from 'redux-mock-store-await-actions';
-import Adapter from 'enzyme-adapter-react-16';
-import { act } from 'react-dom/test-utils';
 import thunk from 'redux-thunk';
 import _ from 'lodash';
-import { Modal } from 'antd';
 import DataProcessingPage from '../../../../../pages/experiments/[experimentId]/data-processing/index';
 
 import generateExperimentSettingsMock from '../../../../test-utils/experimentSettings.mock';
 import { initialPlotConfigStates } from '../../../../../redux/reducers/componentConfig/initialState';
 import initialCellSetsState from '../../../../../redux/reducers/cellSets/initialState';
-
-import { BACKEND_STATUS_LOADING } from '../../../../../redux/actionTypes/backendStatus';
+import initialSamplesState, { sampleTemplate } from '../../../../../redux/reducers/samples/initialState';
 
 import { getBackendStatus } from '../../../../../redux/selectors';
 
-configure({ adapter: new Adapter() });
+import { runPipeline } from '../../../../../redux/actions/pipeline';
+
+jest.mock('../../../../../components/UserButton', () => () => <></>);
+
+// Mock all filter components
+jest.mock('../../../../../components/data-processing/CellSizeDistribution/CellSizeDistribution', () => () => <></>);
+jest.mock('../../../../../components/data-processing/MitochondrialContent/MitochondrialContent', () => () => <></>);
+jest.mock('../../../../../components/data-processing/GenesVsUMIs/GenesVsUMIs', () => () => <></>);
+jest.mock('../../../../../components/data-processing/DoubletScores/DoubletScores', () => () => <></>);
+jest.mock('../../../../../components/data-processing/DataIntegration/DataIntegration', () => () => <></>);
+jest.mock('../../../../../components/data-processing/ConfigureEmbedding/ConfigureEmbedding', () => () => <></>);
```

Comment:
you no longer need to add `.../.../` to these, the linter should understand the relative imports